tensorboard/webapp/runs/views/runs_table/runs_group_menu_button_component.scss

```diff
@@ -36,4 +36,11 @@ limitations under the License.
     height: $_icon-size;
     width: $_icon-size;
   }
+
+  .displayRegexString {
+    padding-left: 40px;
+    .none-set-string {
+      color: gray;
```

Comment:
Please use one of the material color. Better yet, you should use one of the tb-theme method.